tests/src/rules/extensions.js

```diff
@@ -367,6 +368,11 @@ ruleTester.run('extensions', rule, {
           line: 4,
           column: 31,
         },
+        {
+          message: 'Missing file extension for "@/configs/chart"',
```

Comment:
again tho, this should never have a warning when `ignorePackages` is set, because it's a bare idenfier - a package.

Reply:
According to the [scoped packages doc](https://docs.npmjs.com/using-npm/scope.html), `@somescope/somepackagename` is a scoped package, whereas `@/foo/bar` is not.
Publishing `@/foo/bar` as an unscoped package is not allowed, so it's neither a scoped package nor an unscoped package.
![image](https://user-images.githubusercontent.com/28700378/95958875-56a9f800-0e34-11eb-8abe-5a18076c102b.png)

Reply:
I agree that it isn't "a scoped package", but since it doesn't start with `.` or `/`, it is a "bare identifier", which is always "a package" - packages aren't just "publishable packages", they're "anything with a bare identifier".

Reply:
Circling back on this - `@` is a package here (just not a publishable one), so I think this should be ignored when `ignorePackages` is enabled.